www/include/eventLogs/template/viewLog.ihtml

```diff
@@ -66,8 +66,20 @@
         </tr>
         <tr>
             <td class="log-select">{$form.output.html}</td>
-            <td>{$form.StartDate.html}{$form.alternativeDateStartDate.html}{$form.StartTime.html}</td>
-            <td>{$form.EndDate.html}{$form.alternativeDateEndDate.html}{$form.EndTime.html}</td>
+            <td>
+                {$form.StartDate.html}
+                {if isset($form.alternativeDateStartDate)}
```

Comment:
this one is mandatory to be able to localize the date.
Shouldn't you set it if not set ?